From 3523f185bea1336df539148c49ab38968ef7b1c5 Mon Sep 17 00:00:00 2001
From: Nicolas Schier <nicolas.schier@yahoo.no>
Date: Fri, 9 Dec 2011 09:30:25 +0100
Subject: [PATCH] awsetbg: xli is set as fully supported

On Debian squeeze and wheezy xli is working as expected.

Signed-off-by: Nicolas Schier <nicolas.schier@yahoo.no>
---
 utils/awsetbg |    4 ++--
 1 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/utils/awsetbg b/utils/awsetbg
index 48dd178..3466956 100755
--- a/utils/awsetbg
+++ b/utils/awsetbg
@@ -409,8 +409,8 @@ case $WPSETTER in
         tile='-onroot -quiet -border black'
         center='-center -onroot -quiet -border black'
         aspect='-fullscreen -onroot -quiet -border black'
-        full=$aspect #broken
-        debugstory="The fillscreen option (-f) is broken, defaults to (-a). $standardrant"
+        full='-fillscreen -onroot -quiet'
+        debugstory=$standardok
         ;;
     qiv)
         full='--root_s'
-- 
1.7.7.3

